modules/common/deps.sh

```diff
@@ -67,7 +67,7 @@ else
 
     yqpath=bin/yq
     if [ ! -e "$yqpath" ]; then
-        wget "https://github.com/mikefarah/yq/releases/download/2.4.0/yq_${YQ_OS_TYPE}" -O $yqpath
+        wget "https://github.com/mikefarah/yq/releases/download/3.2.1/yq_${YQ_OS_TYPE}" -O $yqpath 
```

Comment:
I remember we needed some new latest feature of `yq` and that's why we bumped the version. I don't remember what and I can't see anything special being called in this PR, maybe we removed the special code in the end. Should I still keep this bump? Won't hurt I guess.

Reply:
I'm fine with the bump ! :+1:

Reply:
the highest version available in Tumbleweed/Leap is 2.10; if we don't need it I wouldn't bump it.

If we do, we need to consume 3.2.1 binary in the Dockerfile, maybe at https://github.com/SUSE/catapult/pull/155.

Reply:
Turns out we need a newer version. With the old one we get this in the cats pod : 


```
│ + cf-operator util template-render                                                                                                                                                          │
│ 2020/04/07 12:03:43 failed to load BOSH deployment manifest /var/run/secrets/resolved-properties/acceptance-tests/properties.yaml: failed to unmarshal BOSH deployment manifest director_uu │
│ id: ""                                                                                                                                                                                      │
│ instance_groups:                                                                                                                                                                            │
│   (name==acceptance-tests):                                                                                                                                                                 │
│     jobs:                                                                                                                                                                                   │
│       (name==acceptance-tests):                                                                                                                                                             │
│         properties:                                                                                                                                                                         │
│           acceptance_tests:                                                                                                                                                                 │
│             include: =internetless                                                                                                                                                          │
│             credhub_mode: "off"                                                                                                                                                             │
│ : error unmarshaling JSON: while decoding JSON: json: cannot unmarshal object into Go struct field Manifest.instance_groups of type manifest.InstanceGroups    
```

Reply:
Seems to be coming from this which doesn't get interpreted by yq: https://github.com/SUSE/catapult/pull/156/files#diff-e363fc0b9655dc80f5abcfaafbe11d82R138